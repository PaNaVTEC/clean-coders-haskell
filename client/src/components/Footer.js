import React, { Component } from 'react'

class Footer extends Component {
  render () {
    return (
      <footer className='footer text-muted'>
        <div className='container'>
          <div className='row'>
            <div className='col mr-auto'>
              <a href='https://codurance.com/' target='_blank' rel='noopener noreferrer'>
                Powered by Codurance.com
              </a>
            </div>
            <div className='col text-right'>
              Crafted with <i className='fa fa-heart text-danger' />
            </div>
          </div>
        </div>
      </footer>
    )
  }
}

export default Footer
