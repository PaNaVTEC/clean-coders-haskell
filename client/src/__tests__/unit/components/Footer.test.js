import Footer from 'components/Footer'

describe('Footer', () => {
  it('renders', () => {
    const wrapper = shallow(<Footer />)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
