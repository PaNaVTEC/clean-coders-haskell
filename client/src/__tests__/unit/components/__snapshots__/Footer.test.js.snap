// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders 1`] = `

<footer class="footer text-muted">
  <div class="container">
    <div class="row">
      <div class="col mr-auto">
        <a href="https://codurance.com/"
           target="_blank"
           rel="noopener noreferrer"
        >
          Powered by Codurance.com
        </a>
      </div>
      <div class="col text-right">
        Crafted with
        <i class="fa fa-heart text-danger">
        </i>
      </div>
    </div>
  </div>
</footer>

`;
